### ISSUE TITLE
FAQ language that all supported LineageOS models are supported should be softened

### ISSUE THREAD

**Issue by Multimath** [lineageos4microg/lineageos4microg.github.io#85]
The FAQ claims that "Can you add support for my device? It is officially supported by LineageOS
We already build LineageOS for microG for every device supported by LineageOS. Find your device in our Downloads page!"

but currently at least one device is missing (miatoll).

The language should be changed to "almost every device" so that readers understand that they are **obligated** to check the Downloads page even if they know that their device is supported by LineageOS.

**Comment by antoinevth** [comment]
As the builds now starts on the first and the 15th of each month (https://github.com/lineageos4microg/docker-lineage-cicd/issues/197#issuecomment-918110759), miatoll hasn't yet been built. It should be built, later today or tomorrow.

**Comment by Multimath** [comment]
Thanks for the info, yes, I see miatoll now. However, the thread which you reference actually **emphasizes** the necessity for changing the language, since it is clear that there could, in the future, very well be devices supported by LineageOS which fail the LineageOS for microG build system (and currently, at least one version for one device does not build stably).

**Comment by antoinevth** [comment]
> BTW, does the information at [lineageos4microg/docker-lineage-cicd#197 (comment)](https://github.com/lineageos4microg/docker-lineage-cicd/issues/197#issuecomment-918110759) mean that the FAQ info that OTA updates occur once a week is now incorrect?

You're right, that's why I proposed a modification (#86 ) of the website but the maintainers haven't comment or merge yet.

**Comment by petefoth** [comment]
Closing this. Accurate information about build frequency and build targets has been added to the wiki and the website
